.github/workflows/deploy.yml

```
@@ -26,7 +26,7 @@ jobs:
         run: |
           npm ci --include=dev
           cp src/configs/config.aws.js src/config.js
-          npm run build
+          npm run build style
```

This passes a positional argument of `style` to `build`, which only works because the command is `run-s`.

```suggestion
          npm run build
          npm run style
```